packages/shared/src/components/profile/UserShortInfo.tsx

```diff
@@ -108,9 +109,20 @@ const UserShortInfoComponent = <Tag extends React.ElementType>(
             )}
             <ReputationUserBadge user={user} />
           </div>
-          <TruncateText className="text-text-secondary" title={`@${username}`}>
-            {transformUsername ? transformUsername(user) : `@${username}`}
-          </TruncateText>
+          <div className="flex text-text-secondary">
+            <TruncateText title={`@${username}`}>
+              {transformUsername ? transformUsername(user) : `@${username}`}
+            </TruncateText>
+
+            {topReader && (
```

Comment:
Re-used again, so component might be good

Reply:
Refactored